src/main/java/com/epam/prd/jjop/helpers/unmodifiablelist_vs_list_of/Main.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.prd.jjop.helpers.unmodifiablelist_vs_list_of;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The purpose of this class is to show the difference between:
+ * <ul>
+ *     <li>{@link java.util.Collections#unmodifiableList(List)}</li>
+ *     <li>{@link List#of()}</li>
```

Comment:
Is this working with CTRL+B click or requires FQN?

Reply:
That's what was automatically converted into by Intellij. I typed in FQN.